lib/util/token_bucket.ex

```diff
@@ -0,0 +1,22 @@
+# Copyright(c) 2015-2018 ACCESS CO., LTD. All rights reserved.
+
+use Croma
+
+defmodule Antikythera.TokenBucket do
+  @moduledoc """
+  A thin wrapper around [`Foretoken`](https://github.com/skirino/foretoken) to avoid collisions between bucket names by prefixing executor pool IDs.
```

Comment:
Insert a blank line between l.7 and l.8.

Reply:
Fixed in 87094bdf3338e52dd4fe86c99edf695f99b3b830